atter-logic/src/commonMain/kotlin/com.attendance.checker/events/viewModels/EventEditViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package com.attendance.checker.events.viewModels
+
+import com.attendance.checker.date.KDate
+import com.attendance.checker.events.Event
+import com.attendance.checker.events.SaveEvent
+import com.attendance.checker.events.presenters.EventEditPresenter
+import com.attendance.checker.shared.viewModels.ViewModel
+import com.attendance.checker.uuid.KUUID
+
+class EventEditViewModel constructor(private val presenter: EventEditPresenter,
```

Comment:
Did you try to use: https://github.com/florent37/Multiplatform-LiveData to avoid having a`*Presenter` class? 
(Which by the way I'd call `EventEditView`) :P

Reply:
Don't work (as discussed on slack)